### PR TITLE
INFRA18 - scaffold Firebase Functions

### LINKED ISSUE #149: INFRA18 - scaffold Firebase Functions
The structure of the scaffolding should respect this scheme: https://medium.com/@george43g/organise-firebase-functions-part-1-optimize-cold-start-time-cd36f76d2133

### PR DESCRIPTION
**Pull request by gregoriopalama**
Closes #149 .

### CODE REVIEW

File: functions/src/api/auth/on-create.ts
Comment (irsooti) [inline functions/src/api/auth/on-create.ts:1]: 🤔

File: .firebaserc [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
   "projects": {
-    "default": "devfestpescara-2023"
+    "default": "hedwig-demo"
```
Comment (irsooti) [inline .firebaserc:3]: Sicuri che vogliamo lasciare questo default?

Comment (gregoriopalama) [reply .firebaserc:3]: Better add the .firebaserc to the gitignore?

File: functions/src/api/auth/on-create.ts
Comment (gregoriopalama) [reply functions/src/api/auth/on-create.ts:1]: oops... forgot to delete this. done

File: .firebaserc (hunk 1 above)
Comment (irsooti) [reply .firebaserc:3]: mmmh dunno, is it something required I guess... ok, maybe we could leave it there, up to you in this case 😆

**Pull request merged**